build/log.go

```diff
@@ -36,6 +36,21 @@ func (t LogType) String() string {
 	}
 }
 
+// logCompressors maps the identifier for each supported compression algorithm
+// to the extension used for the compressed log files.
+var logCompressors = map[string]string{
+	"gzip": "gz",
```

Comment:
Should we export these constants (just the names) here so other projects using the `LogRotator` struct can pass in a constant instead of a string?